actors/runtime/src/test_utils.rs

```diff
@@ -62,10 +62,18 @@ lazy_static! {
 
 const IPLD_RAW: u64 = 0x55;
 
+/// Returns an identify CID for bz.
 fn make_builtin(bz: &[u8]) -> Cid {
     Cid::new_v1(IPLD_RAW, Code::Identity.digest(bz))
 }
 
+// Calls a function and asserts the result is an ActorError with specified exit code.
+pub fn expect_error<T>(code: ExitCode, f: impl FnOnce() -> Result<T, ActorError>) {
```

Comment:
we are having the same discussion in #60, and both steven and i think the closure is redundant here; we can just pass the Result directly.

Reply:
I'll steal that nice generic T in Result<T, ActorError> though

Reply:
Hmm, I haven't checked all the call sites in Go, but I'd be a bit surprised if none have more than a single line. Nevertheless, zen's version is better than mine and until we need the closure, happy to drop it.